zerver/models.py

```diff
@@ -1827,3 +1829,20 @@ def get_bot_services(user_profile_id):
 def get_service_profile(email, realm, service_name):
     # type: (str, Realm, str) -> Service
     return Service.objects.get(user_profile__email=email, user_profile__realm=realm, name=service_name)
+
+def get_slash_commands(realm, is_active=None):
+    # type: (Realm, Optional[bool]) -> List[Any]
+    if is_active is None:
+        users = list(UserProfile.objects.filter(realm=realm, bot_type=UserProfile.SLASH_COMMANDS).values('email'))
+    else:
+        users = list(UserProfile.objects.filter(realm=realm, bot_type=UserProfile.SLASH_COMMANDS,
+                                                is_active=is_active).values('email'))
+    slash_commands = []
+    for user in users:
+        slash_commands.append("/" + email_to_username(user['email']))
```

Comment:
Would it be possible to decide (and document) when and where the leading slash will be included in strings, and when it will not?

My personal preference would be to ONLY consider the leading slash when it's part of a message (i.e. during regex matching), but to remove it everywhere else.  However, I would be okay with other options as long as they are clear, consistent, and documented somewhere.  My main goal here is to avoid the sorts of errors / confusion that arise when there's an extra slash or a missing slash in a string.